src/mocks/browser.js

```diff
@@ -0,0 +1,11 @@
+// src/mocks/browser.js
+import { setupWorker } from './msw.js'
+// import { setupWorker } from '@bundled-es-modules/msw'
+// import '@bundled-es-modules/msw/lib/iife/index.js';
+import { handlers } from './handlers.js'
+
+// This configures a Service Worker with the given request handlers.
+// export const worker = MockServiceWorker.setupWorker(...handlers)
+export const worker = setupWorker(...handlers)
+await worker.start({ serviceWorker: { url: '/mockServiceWorker.js' } });
```

Comment:
MSW for `@web/test-runner`
To be included into unit test file
a
```js
import "../src/mocks/browser.js";
```
Alternatively is to add it as import into all tests hook